.github/workflows/pull-request.yaml

```
@@ -12,6 +12,7 @@ on:
       - 'sec-scanners-config.yaml'
     branches:
       - main
+      - release-**
```

```suggestion
      - "release-**"
```

---

👍 I think this is even better
```suggestion
      - 'release-**'
```